webpack/ForemanTasks/Components/TasksTable/Components/__test__/CancelResumeConfirm.test.js

```diff
@@ -6,8 +6,7 @@ import { CANCEL, RESUME, CLOSED } from '../../TasksTableConstants';
 const baseProps = {
   closeModal: () => null,
   selectedRowsLen: 3,
-  action: () => null,
-  selected: [1, 2, 3],
+  action: { CANCEL: () => null, RESUME: () => null },
```

Comment:
Why not
```suggestion
  action: { CANCEL: jest.fn(), RESUME: jest.fn(), },
```
just for consistency.